modules/templates/util_string.go

```diff
@@ -36,3 +37,8 @@ func (su *StringUtils) Join(a []string, sep string) string {
 func (su *StringUtils) EllipsisString(s string, max int) string {
 	return base.EllipsisString(s, max)
 }
+
+func (su *StringUtils) ToInt(s string) int64 {
```

Comment:
I think it doesn't need this new function.

It could make `Sec2Time` to accept `any` and call "util.ToInt64"

```
func SecToTime(dur any) string {
	duration, _ := util.ToInt64(dur)
	// ... the old code ...
```

Then the template can be as simple as : `{{.Content|Sec2Time}}`

Reply:
done